clients/pkg/promtail/targets/journal/journaltarget.go

```diff
@@ -174,12 +174,26 @@ func journalTargetWithReader(
 		return nil, errors.Wrap(err, "parsing journal reader 'max_age' config value")
 	}
 
-	cfg := t.generateJournalConfig(journalConfigBuilder{
+	cb := journalConfigBuilder{
 		JournalPath: targetConfig.Path,
 		Position:    position,
 		MaxAge:      maxAge,
 		EntryFunc:   entryFunc,
-	})
+	}
+
+	matches := strings.Fields(targetConfig.Matches)
```

Comment:
Can you please add some tests for this @carlospeon?

Reply:
Sure, tests added.

Reply:
Thank you!